src/sdk/sql_insert_row.cc

```diff
@@ -198,6 +198,9 @@ bool SQLInsertRow::AppendInt64(int64_t val) {
 }
 
 bool SQLInsertRow::AppendTimestamp(int64_t val) {
+    if (val < 0) {
+        return false;
+    }
```

Comment:
add check in `AppendInt64`

Reply:
should we guarantee `int64_t val` in `AppendInt64` be non-negative?

Reply:
guarantee no negative if the col_id in `ts_set_`